src/lib/CMakeLists.txt

```diff
@@ -441,12 +441,16 @@ if (${ENABLE_JIT_SUPPORT})
     include(${CMAKE_SOURCE_DIR}/cmake/EmbedLLVM.cmake)
     set(JIT_EMBEDDED_SOURCES
         # Files listed here will be embedded into the binary as LLVM-IR
-        operators/jit_operator/jit_operations.cpp
-        operators/jit_operator/jit_operations.hpp
+        abstract_expression.cpp
```

Comment:
"Coverage — decreased by 1.7 compared to master and is now 88.4% " - the coverage report looks wrong. Any idea what happened there?

Reply:
I think coverage is computed using `gcc`? When compiling with `gcc`, `ENABLE_JIT_SUPPORT` is `false` and most of the JIT files are not included in the built and thus not counted. But maybe they still count towards the total number of lines?